### ISSUE TITLE
[CreateIndex]:Create index failed: index already exists. 

### ISSUE THREAD

**Issue by yongpengli-z** [milvus-io/milvus-sdk-java#311]
explain：
I created index first.then I dropped index and response success. then I create index again.

steps:
1.create collection
2.create index
3.drop index
4.create index

**Comment by yhmo** [comment]
@yongpengli-z 

I could not reproduce this with latest master sources code:

      ```
  String collectionName = "aaa";
        // collection schema
        String field1Name = "long_field";
        String field2Name = "vec_field";
        List<FieldType> fieldsSchema = new ArrayList<>();
        fieldsSchema.add(FieldType.newBuilder()
                .withPrimaryKey(true)
                .withAutoID(false)
                .withDataType(DataType.Int64)
                .withName(field1Name)
                .withDescription("identity")
                .build());

        fieldsSchema.add(FieldType.newBuilder()
                .withDataType(DataType.FloatVector)
                .withName(field2Name)
                .withDescription("face")
                .withDimension(dimension)
                .build());

        // create collection
        CreateCollectionParam createParam = CreateCollectionParam.newBuilder()
                .withCollectionName(randomCollectionName)
                .withDescription("test")
                .withFieldTypes(fieldsSchema)
                .build();

        R<RpcStatus> createR = client.createCollection(createParam);
        assertEquals(R.Status.Success.getCode(), createR.getStatus().intValue());
        
        // create index
        CreateIndexParam indexParam = CreateIndexParam.newBuilder()
                .withCollectionName(randomCollectionName)
                .withFieldName(field2Name)
                .withIndexType(IndexType.IVF_FLAT)
                .withMetricType(MetricType.L2)
                .withExtraParam("{\"nlist\":256}")
                .withSyncMode(Boolean.TRUE)
                .withSyncWaitingInterval(500L)
                .withSyncWaitingTimeout(30L)
                .build();

        R<RpcStatus> createIndexR = client.createIndex(indexParam);
        assertEquals(R.Status.Success.getCode(), createIndexR.getStatus().intValue());

        R<RpcStatus> kk = client.dropIndex(DropIndexParam.newBuilder()
                        .withCollectionName(randomCollectionName)
                        .withFieldName(field2Name)
                        .build());

        createIndexR = client.createIndex(indexParam);
        assertEquals(R.Status.Success.getCode(), createIndexR.getStatus().intValue());
```

**Comment by yongpengli-z** [comment]
JAVA-SDK 2.1.0 is ok！
![image](https://user-images.githubusercontent.com/103410837/177110014-091e2167-f15e-4724-af1c-ab1e2ff2bed5.png)

**Comment by yongpengli-z** [comment]
[Issue311.java.zip](https://github.com/milvus-io/milvus-sdk-java/files/9044392/Issue311.java.zip)
![image](https://user-images.githubusercontent.com/103410837/177277805-0fd8c5f8-74d2-4288-9260-dedfefd27189.png)

**Comment by yhmo** [comment]
Fixed by #326
The server index behavior has been changed.
For createIndex(), user input collection name, field name, and index name to define an index. If index name not provided, it will be default value "_default_idx".
For describeIndex()/dropIndex()/getIndexState()/getIndexBuildProgress(), user input collection name and index name. No need to input field name.

**Comment by yongpengli-z** [comment]
Verify passed by 2.1.0-beta2
